src/algebra/add_torsor.lean

```diff
@@ -72,87 +60,23 @@ class add_torsor (G : out_param Type*) (P : Type*) [out_param $ add_group G]
 (vadd_vsub' : ∀ (g : G) (p : P), g +ᵥ p -ᵥ p = g)
 
 attribute [instance, priority 100, nolint dangerous_instance] add_torsor.nonempty
-
-end old_structure_cmd
+attribute [nolint dangerous_instance] add_torsor.to_has_vsub
```

Comment:
Why did this now appear?

Reply:
I don't know. This may be related to the fact that `add_action` and `add_torsor` don't use `old_structure_cmd` anymore.